pkg/kv/kvserver/rangefeed/budget.go

```diff
@@ -16,11 +16,26 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/cockroachdb/cockroach/pkg/roachpb"
+	"github.com/cockroachdb/cockroach/pkg/rpc"
+	"github.com/cockroachdb/cockroach/pkg/util/envutil"
 	"github.com/cockroachdb/cockroach/pkg/util/mon"
 	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
 	"github.com/cockroachdb/errors"
 )
 
+// useBudgets controls if RangeFeed memory budgets are enabled. Overridable by
+// environment variable.
+var useBudgets = envutil.EnvOrDefaultBool("COCKROACH_USE_RANGEFEED_MEM_BUDGETS", true)
+
+// maxFeedFraction is maximum percentage of feed memory pool that could be
+// allocated to a single feed budget. Overridable by environment variable.
+var maxFeedFraction = envutil.EnvOrDefaultFloat64("COCKROACH_RANGEFEED_MEM_FRACTION", 0.1)
+
+// Pre allocated memory limit for system RangeFeeds.
+var systemRangeFeedBudget = envutil.EnvOrDefaultInt64("COCKROACH_RANGEFEED_SYSTEM_BUDGET",
```

Comment:
Should we bump this a bit higher, just to give us some headroom beyond the Raft max command size? Have you done any stress testing with e.g. zone config updates or other system rangefeeds?